[BUG] Exceptions caused in FileDataStorageManager

Google Play trace:

```
java.lang.RuntimeException: 
  at android.os.AsyncTask$4.done (AsyncTask.java:399)
  at java.util.concurrent.FutureTask.finishCompletion (FutureTask.java:383)
  at java.util.concurrent.FutureTask.setException (FutureTask.java:252)
  at java.util.concurrent.FutureTask.run (FutureTask.java:271)
  at android.os.AsyncTask$SerialExecutor$1.run (AsyncTask.java:289)
  at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1167)
  at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:641)
  at java.lang.Thread.run (Thread.java:919)Caused by: java.lang.IllegalArgumentException: 
 
Caused by: java.lang.IllegalArgumentException: 
  at com.owncloud.android.datamodel.FileDataStorageManager.<init> (Unknown Source:7)
  at com.owncloud.android.files.services.AvailableOfflineSyncJobService$AvailableOfflineJobTask.doInBackground (AvailableOfflineSyncJobService.java:76) 
  at com.owncloud.android.files.services.AvailableOfflineSyncJobService$AvailableOfflineJobTask.doInBackground (AvailableOfflineSyncJobService.java:61) 
  at android.os.AsyncTask$3.call (AsyncTask.java:378) 
  at java.util.concurrent.FutureTask.run (FutureTask.java:266)
```

Android app 2.15.1
Android 10

--- a/owncloudApp/src/main/java/com/owncloud/android/files/services/AvailableOfflineSyncJobService.java
+++ b/owncloudApp/src/main/java/com/owncloud/android/files/services/AvailableOfflineSyncJobService.java
@@ -72,6 +72,11 @@ public class AvailableOfflineSyncJobService extends JobService {
             Account account =
                     AccountUtils.getCurrentOwnCloudAccount(mAvailableOfflineJobService.getApplicationContext());
 
+            if (mAvailableOfflineJobService.getContentResolver() == null) {
+                Timber.w("Content resolver is null, do not sync av. offline files.");
+                return jobParams[0];
+            }
+
             FileDataStorageManager fileDataStorageManager = new FileDataStorageManager(
                     mAvailableOfflineJobService, account, mAvailableOfflineJobService.getContentResolver()
             );

